### ISSUE TITLE
Two consistent failures on 2.1.6

### ISSUE THREAD

**Issue by todb-r7** [packetfu/packetfu#87]
```
Finished tests in 0.006638s, 1205.0986 tests/s, 2410.1972 assertions/s.

  1) Failure:
EthTest#test_ethmac [test_eth.rb:15]:
<"\u001At\xDE"> expected but was
<"\x1At\xDE">.

  2) Failure:
EthTest#test_macaddr [test_eth.rb:50]:
<"\u0000\u0003/\u001At\xDE"> expected but was
<"\x00\x03/\x1At\xDE">.

8 tests, 16 assertions, 2 failures, 0 errors, 0 skips

ruby -v: ruby 2.1.6p336 (2015-04-13 revision 50298) [x86_64-linux]
--------------------------------------------------------------------------------
[ruby-2.1.6@packetfu](master) 
todb@mazikeen:~/git/packetfu/test$ 

```

**Comment by todb-r7** [comment]
Looks like it's a simple string encoding deal, missing a magic header, hunting down shortly.

**Comment by claudijd** [comment]
Fixed in #88
